src/Type/Row.purs

```diff
@@ -43,6 +47,8 @@ foreign import kind RowList
 foreign import data Nil :: RowList
 foreign import data Cons :: Symbol -> Type -> RowList -> RowList
 
+data RLProxy (rowList :: RowList) = RLProxy
```

Comment:
Do you think it'll be useful to have value-level equivalents for `RowToList` and `ListToRow`?